src/Microsoft.Extensions.DependencyInjection.Specification.Tests/DependencyInjectionSpecificationTests.cs

```diff
@@ -590,7 +562,7 @@ public void AttemptingToResolveNonexistentServiceReturnsNull()
         }
 
         [Fact]
-        public void NonexistentServiceCanBeIEnumerableResolved()
+        public void NonexistentServiceCanNotBeIEnumerableResolved()
```

Comment:
Should add a variant that says service not added as enumerable cannot be IEnumerable resolved